Signal/src/util/OWSScreenLockUI.m

```diff
@@ -332,9 +427,23 @@ - (void)registrationStateDidChange
     [self ensureScreenProtection];
 }
 
+- (void)clearAuthUIWhenActive
+{
+    if (self.appIsInactive) {
```

Comment:
For continuity, continue to present blocking screen in "screen lock" mode while dismissing the "local auth UI".